unicef/giga-dagster
# feat: add admin__create_lakehouse_local_job

## What type of PR is this?

- `feat`: Commits that add a new feature

## Summary

Implements a utility job `admin__create_lakehouse_local_job` that jumpstarts the creation of a lakehouse in ADLS for local development.

## How to test

1. Check if the folder `lakehouse-local-[username]` exists in the dev blob container. If it does, temporarily rename or delete the folder.
2. Set `LAKEHOUSE_USERNAME` in `.env`
3. Run the `admin__create_lakehouse_local_job` in Dagster.
